contracts/deployables/erc20/VotesERC20LockableV1.sol

```diff
@@ -101,11 +102,15 @@ contract VotesERC20LockableV1 is IVotesERC20LockableV1, VotesERC20V1 {
         _mint(to, amount);
     }
 
+    function burn(address account, uint256 amount) external onlyOwner {
+        _burn(account, amount);
+    }
```

Comment:
As @jigglyjams said, let's remove `onlyOwner`, remove the `account` param, and call `_burn(msg.sender, amount)`